### PR TITLE
BufNMFCross reference

### PR DESCRIPTION
**Pull request by tedmoore**

### CODE REVIEW

File: src/routes/reference/bufnmfcross.svx [hunk 1]
```diff
@@ -9,4 +9,73 @@ tags:
...
+
+<Audio
+src="/reference/bufnmfcross/output.mp3"
+label="Output of BufNMFCross using the drums as the target and the synth sounds as the source."
+waveform={true}
+/>
+
+Rather than replacing single spectral frames in the target with single spectral frames in the source, BufNMFCross looks for opportunities to use a sequence of spectral frames from the source buffer to enhance the perception of the timbral morphology of both the source and target. This functionality can be adjusted using the ``continuity`` argument.
+
+Additionally, BufNMFCross will avoid repeating a spectral frame within a specified duration of time (using the argument ``timeSparsity``). This helps avoid a common problem with this type of audio mosaiking: the repetition or over use of a single sound element from the source.
```
Comment (tremblap) [inline src/routes/reference/bufnmfcross.svx:65]: over use -> overuse

one could talk of a similar approach to round-robin in modern samplers?

File: src/routes/reference/bufnmfcross.svx [hunk 2]
```diff
@@ -9,4 +9,73 @@ tags:
...
+
+Finally, BufNMFCross allows for a spectral frame in the ``target`` to be approximated using more than one spectral frame from the ``source``. This can help more closely approximate the spectral frames in the target, however, too using too many ``source`` spectral frames at once can cause phase cancellation or other undesirable artefacts. This functionality can be adjusted using the ``polyphony`` argument, which sets the maximum number of ``source`` spectral frames that can be used at one time.
```
Comment (tremblap) [inline src/routes/reference/bufnmfcross.svx:67]: too using too many -> using too many

File: src/routes/reference/bufnmfcross.svx [hunk 3]
```diff
@@ -9,4 +9,73 @@ tags:
...
+
+<IndentNote type="pointer">
+
+In contrast to most uses of [BufNMF](/reference/bufnmf), each spectral frame of ``source`` is a spectral template. NMF is ``O(N^2)`` where ``N`` is the number of spectral templates, so longer ``source`` buffers will take dramatically longer to process.
```
Comment (tremblap) [inline src/routes/reference/bufnmfcross.svx:71]: maybe big O notation is a bit obscure. @weefuzzy is not a fan usually. maybe taming it down by saying it in words? like doubling the size of source will quadruple the computation time.

**Review by tremblap — CHANGES_REQUESTED** [review]
almost perfect. a few suggestions of typos and wordings

File: src/routes/reference/bufnmfcross.svx (hunk 1 above)
Comment (tedmoore) [reply src/routes/reference/bufnmfcross.svx:65]: > one could talk of a similar approach to round-robin in modern samplers? 
nice one. added that in.

**Pull request merged**